apps/editing-toolkit/editing-toolkit-plugin/wpcom-universal-themes/index.php

```diff
@@ -9,11 +9,45 @@
 
 namespace A8C\FSE;
 
+use Automattic\Jetpack\Connection\Client;
+
 /**
  * This is the option name for enabling/disabling.
  */
 define( 'ACTIVATE_FSE_OPTION_NAME', 'wpcom_is_fse_activated' );
 
+/**
+ * Checks if site is eligible for Core FSE
+ *
+ * As of 2021, this is only sites for existing users who explicitly
+ * choose to enroll in the WP.com FSE Beta
+ */
+function is_core_fse_eligible() {
```

Comment:
Similar to the comment in the diff, I think we can start dropping the `core` part when naming FSE things.